roc/log.go

```diff
@@ -181,6 +192,36 @@ func loggerRoutine() {
 	}
 }
 
+func getGID() uint64 {
+	b := make([]byte, 64)
+	b = b[:runtime.Stack(b, false)]
+	b = bytes.TrimPrefix(b, []byte("goroutine "))
+	b = b[:bytes.IndexByte(b, ' ')]
+	n, _ := strconv.ParseUint(string(b), 10, 64)
+	return n
+}
+
+func logWrite(level LogLevel, text string) {
+	if level >= LogDebug {
+		module, ok := debug.ReadBuildInfo()
+		_, file, line, callerOk := runtime.Caller(0)
+
+		if ok && callerOk {
```

Comment:
I think we can log message even if callerOk is false. Missing file and line is not that critical.

Reply:
test done